sw/host/opentitantool/src/command/rsa.rs

```diff
@@ -0,0 +1,189 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use erased_serde::Serialize;
+use std::any::Any;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+use opentitanlib::app::command::CommandDispatch;
+use opentitanlib::app::TransportWrapper;
+use opentitanlib::crypto::rsa::{
+    Exponent, Modulus, N0Inv, RsaPrivateKey, RsaPublicKey, Signature, RR,
+};
+use opentitanlib::crypto::sha256::Sha256Digest;
+use opentitanlib::util::parse_int::ParseInt;
+
+#[derive(serde::Serialize)]
+pub struct RsaKeyInfo {
+    pub key_num_bits: usize,
+    pub modulus: Modulus,
+    pub public_exponent: Exponent,
+    pub n0_inv: N0Inv,
+    pub rr: RR,
+}
+
+#[derive(serde::Serialize)]
+pub struct RsaKeyInfoInWords {
+    pub key_num_bits: usize,
+    pub modulus: Vec<String>,
+    pub public_exponent: Vec<String>,
+    pub n0_inv: Vec<String>,
+    pub rr: Vec<String>,
+}
+
+/// Show public information of a private or public RSA key
+#[derive(Debug, StructOpt)]
+pub struct RsaKeyShowCommand {
+    #[structopt(
+        name = "DER_FILE",
+        help = "RSA public or private key file in DER format"
+    )]
+    der_file: PathBuf,
+}
+
+impl CommandDispatch for RsaKeyShowCommand {
```

Comment:
Just curious: This doesn't have to go into this PR, but I was wondering if it's relatively easy to add an option to format the output like [test_key_0_rsa_3072_exp_f4.h](https://github.com/lowRISC/opentitan/blob/master/sw/device/silicon_creator/mask_rom/keys/test_key_0_rsa_3072_exp_f4.h).

Reply:
SG!